### DISCUSSION TITLE
chaftrix: image animation + matrix effect in terminal window (c, chafa)

### DISCUSSION THREAD

**Discussion by christosangelopoulos** [hpjansson/chafa#227] [Show and tell]
[https://gitlab.com/christosangel/chaftrix](https://gitlab.com/christosangel/chaftrix)

This program written in **C** will render the **matrix effect** in the terminal window in the background, while rendering an **image** in the foreground, allowing **animation of this image in one or two dimensions**.

[![video.png](https://gitlab.com/christosangel/chaftrix/-/raw/main/video.png)](https://www.youtube.com/watch?v=fUZXQDAQJ1w)

**Comment by AnonymouX47** [comment]
I love this so much!

Great work! Well done.

**Reply by christosangelopoulos** [reply]
Thanks!

**Comment by clort81** [comment]
I am curious where i can modify the parameters passed to the chafa invocation.
 char *system0="chafa --symbols block+border-space-wide-inverted "
ah embarassing right at the top.

**Comment by hpjansson** [comment]
Love this!
